cdap-securestore-ext-gcp-secretstore/pom.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright © 2019 Cask Data, Inc.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+  ~ use this file except in compliance with the License. You may obtain a copy of
+  ~ the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+  ~ WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+  ~ License for the specific language governing permissions and limitations under
+  ~ the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>cdap</artifactId>
+    <groupId>io.cdap.cdap</groupId>
+    <version>6.8.0</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>cdap-securestore-ext-gcp-secretstore</artifactId>
+  <name>CDAP Google Cloud Secret Store Secure Store Extension</name>
+  <packaging>jar</packaging>
+
+  <properties>
+    <google.cloud.secretmanager.version>2.8.0</google.cloud.secretmanager.version>
+    <google.api.client.version>1.34.0</google.api.client.version>
+    <google.cloud.libraries.bom.version>26.1.4</google.cloud.libraries.bom.version>
+    <guava.version>25.1-jre</guava.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-securestore-spi</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-secretmanager</artifactId>
+      <version>${google.cloud.secretmanager.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>google-cloud-core</artifactId>
+      <!-- TODO(bhorsley): Should a version be specified? -->
+    </dependency>
+    <dependency>
+      <!-- TODO(bhorsley): is this necessary? Seemed to be needed to not grab 2.5.0 version of proto -->
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>proto-google-cloud-secretmanager-v1</artifactId>
+      <version>${google.cloud.secretmanager.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>${guava.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api-client</groupId>
```

Comment:
api-client and http-client both needed to be specified directly?

Reply:
got rid of them, you're right.

had to do a lot of playing with this until I discovered that the published secret manager libraries were out of date.